_plugins/gtn/boxify.rb

```diff
@@ -64,6 +64,19 @@ module Boxify
         "hands_on" => "Práctica",
         "question" => "Preguntas",
         "tip" => "Consejo",
+      },
+      "fr" => {
+        "agenda" => "Agenda",
+        "code-in" => "Input",
+        "code-out" => "Output",
+        "comment" => "Comment",
+        "details" => "Details",
+        "hands-on" => "Hands-on",
+        "hands_on" => "Hands-on",
+        "question" => "Question",
+        "solution" => "Solution",
+        "tip" => "Tip",
+        "warning" => "Warning",
```

Comment:
@yvanlebras: we just need some french translations for the boxtitles

Reply:
Done! THANK YOU Saskia !